.github/workflows/_release-pypi.yml

```diff
@@ -20,7 +25,7 @@ jobs:
       - name: Setup Python for PyPI upload
         uses: actions/setup-python@v5
         with:
-          python-version: '3.14'
+          python-version: ${{ inputs.python_version }}
```

Comment:
for uploading to PyPI, again same logic. use the Python version passed from the main build-wheel-upload .yml file